### ISSUE TITLE
La Guerre Des Monders Season 1 Episode 4

### ISSUE THREAD

**Issue by Nemesh64** [justdan96/tsMuxer#724]
Hello,

La Guerre Des Mondes Season 1 Disc 2 episode 4.
BDInfo shows both audiotracks (Frensh & English) as DTS-HD 5.1 audio [LINK](https://imgur.com/0h3WAcs)
Also, PowerDVD shows these tracks as DTS-HD 51.1 Audio.
However, as I load the playlist from this epiosde (playlist 0008) into
TsMuxer (version from 28-06-20230) it shows the tracks as LCPM 2.0: [LINK](https://imgur.com/jZweLY0)
If I try to mux these tracks I get a error from, TsMuxer: AV frame too large (32770 bytes). Please increase AV buffer: [LINK](https://imgur.com/3CxLsUW)
No matter how large I make the buffer, TsMuxer will not mux.
Is this a bug in TsMuxer?
With regards,

Nemesh

**Comment by jcdr428** [comment]
@Nemesh64 without having the files it is difficult to know.
Do you have the same results with tsMuxer 2.6.12 ?

**Comment by Nemesh64** [comment]
Hello Dan,

Many thanks for the fast reply.
Have send you a email to you gmail account.
Please let me know if you have received it.
With kind regards,

Nemesh64

**Comment by Hellboy00000** [comment]
I haved a similar problem with https://www.blu-ray.com/movies/Halloween-4K-Blu-ray/294501/. If i remember correctly, the problem was that the playlist contain 2 m2ts. The first with AC3 and the second with TrueHD. And when added the playlist, tsMuxer select the audio from the first m2ts. So used the second m2ts and muxed correctly. Maybe is the same problem.

**Comment by Nemesh64** [comment]
Hello Dan,

Opened the same playlist (0008) with TsMuxer 2.6.12.
The result is the same: [LINK](https://imgur.com/i8wWqt6)
If you want some files for evaluiation purposes, please let me know.
With regards,

Nemesh

**Comment by jcdr428** [comment]
@Nemesh64 please PM me the links on doom9.org user jcdr428.

**Comment by Nemesh64** [comment]
The links in the above comments?

**Comment by jcdr428** [comment]
The links for downloading  the BDMV samples allowing to reproduce the bug

**Comment by Nemesh64** [comment]
No Dan, it says that it is LCPM 2.0 but no audio.
Will upload it with the .mpls.
Link on Doom9.

**Comment by Nemesh64** [comment]
Have send you a PM with link on Doom9.
Can you please let me know trhat you have received the PM?
With regards,

Nemesh

**Comment by jcdr428** [comment]
@Nemesh64 the bug comes from the fact that the intro m2ts contains two LPCM audios.
tsMuxer should discard the intro when streams are differents from the other m2ts, this is still work to be done.

**Comment by Nemesh64** [comment]
Many thanks for the support Dan.
With kind regards,

Nemesh

**Comment by Nemesh64** [comment]
Hello Dan,

Today Nightly build from 2023-09-30-01-50-24

Fix LPCM header

Tried it right away.
However, TsMuxer still loads the LCPM audio instead of the DTS-HD MA audio track.
Have made a screenshot: https://imgur.com/s6BorIV

**Comment by jcdr428** [comment]
@Nemesh64 yes tsMuxer still gets the streams from the first play in the playlist.
The issue will remain open until somebody has pushed a fix.

**Comment by Nemesh64** [comment]
So that I understand it correctly, you can't fix the issue?

**Comment by jcdr428** [comment]
@Nemesh64 tsMuxer is an open source project with no dedicated developers. Everybody is free to propose enhancements and fixes.
Personally, I might or might not look at the subject in the future -hopefully somebody else will do.

**Comment by Nemesh64** [comment]
Thanks jcdr, all clear.
Hope it will be fixed in the near future so that I can finish these 2 seasons.

**Comment by Nemesh64** [comment]
The movie Maneater from 2022 has the same issue.
